SEED-platform/seed
# Chore/bsync scenario savings

#### Any background context you want to provide?
#### What's this PR do?
Adds scenario annual savings to the inventory detail page. Also proper handling of BSync meter data for property_meters and _build_column_def.
#### How should this be manually tested?
Navigate to the property details page of a property that has scenarios with savings data and verify it's there.
#### What are the relevant tickets?
#1899 
#### Screenshots (if appropriate)
<img width="1353" alt="Screen Shot 2020-02-26 at 11 41 37 AM" src="https://user-images.githubusercontent.com/18518728/75376287-070ac200-588d-11ea-97cb-9b4cac77451f.png">
